### PR TITLE
boss-lite -> snowwhite && quick code review.

### PR DESCRIPTION
**Pull request by wayneeseguin**
I also added a quick MIT LICENSE file, better than nothing :)

### CODE REVIEW

File: installer.sh [hunk 1]
```diff
@@ -54,13 +58,27 @@ fi
...
-echo
-echo "READ: the following document for using bosh-lite with AWS"
-echo https://github.com/cloudfoundry/bosh-lite/blob/master/docs/aws-provider.md
-echo
-echo "NOTE: this is a PRIVATE repository that has your AWS credentials in it"
-echo
+git commit -m "Initial scaffold for snowwhite project."
+
+echo <<EOS
+Congrats! You're snowwhite project is ready!
```
Comment (drnic) [inline installer.sh]: Your

@fearoffish

File: scaffold/README.md [hunk 2]
```diff
@@ -5,32 +5,32 @@ This repository includes scripts to create [bosh-lite](https://github.com/cloudf
...
+snowwhite dir drnic
+snowwhite ip drnic
+snowwhite list
+snowwhite ssh drnic
+snowwhite destroy drnic [-f]
+snowwhite update
 ```
 
-Read the [Usage instructions](https://github.com/cloudfoundry-community/boss-lite#usage) for more information; especially after updating.
+Read the [Usage instructions](https://github.com/cloudfoundry-community/#usage) for more information; especially after updating.
```
Comment (drnic) [inline scaffold/README.md:23]: fix url

File: scaffold/README.md [hunk 3]
```diff
@@ -5,32 +5,32 @@ This repository includes scripts to create [bosh-lite](https://github.com/cloudf
...
 
 Global CLI
 ----------
 
-You can use the `boss-lite` CLI from anywhere if you add this project's `bin` into your `$PATH`.
+You can use the `` CLI from anywhere if you add this project's `bin` into your `$PATH`.
```
Comment (drnic) [inline scaffold/README.md]: `snowwhite` missing

File: scaffold/bin/boss-lite [hunk 4]
```diff
@@ -1,231 +0,0 @@
-#!/bin/bash
```
Comment (drnic) [inline scaffold/bin/boss-lite:1]: neat

Comment (drnic) [inline scaffold/bin/boss-lite:1]: would this script need to change into the local folder first?

Comment (drnic) [inline scaffold/bin/boss-lite:1]:
```
DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+cd ${DIR}
```

File: installer.sh (hunk 1 above)
Comment (wayneeseguin) [inline installer.sh]: LOL! <3

File: scaffold/README.md (hunk 3 above)
Comment (wayneeseguin) [inline scaffold/README.md]: good catch!

File: scaffold/bin/boss-lite (hunk 4 above)
Comment (wayneeseguin) [inline scaffold/bin/boss-lite:1]: I'm not following this last question...

**Comment by drnic** [comment]
Thanks!

**Pull request merged**